internal/tests_config.go

```diff
@@ -102,6 +154,29 @@ type CompareResult struct {
 	Stats          Stats
 }
 
+// Compiles result output with expected outputs and return expected status.
```

Comment:
I think, even for non-exported functions we keep the "godoc"-like format 
```suggestion
// getExpectedStatusRegex compiles result output with expected outputs and return expected status.
```